css/style.css

```diff
@@ -0,0 +1,700 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+:root {
+  --clr-dark: #272a31;
+  --clr-red: #ec5242;
+  --clr-gray: #ececec;
```

Comment:
- [ ] It is great that you are using CSS variables, it helps a lot with the organization of the code. But the color swatch provided by the template has one little difference with the lighter gray color, the one provided by the template is the `#d3d3d3` color when you used `#ececec`. They are very similar so I ask you to change that so your project matches the color swatch provided by the template.